week2/homework/src/index.js

```diff
@@ -1,3 +1,52 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+const program = require('commander');
+const commands = require('./commands');
+const addToDoList = require('./addToDos');
+const showList = require('./showList');
+const remove = require('./removeList');
+const reset = require('./resetList');
+if (process.argv.length === 2) {
+  commands.showCommandsList();
+} else {
+  program
+    .command('help')
+    .alias('hp')
+    .description('Shows list of commands')
+    .action(() => {
+      commands.showCommandsList();
+    });
+
+  program
+    .command('add <toDo>')
+    .alias('a')
+    .description('Adds a to-do item')
+    .action(toDo => {
+      addToDoList.generateToDosList(toDo);
```

Comment:
Here also it `Adds a to-do item` your method called `addToDoList.generateToDosList`.